shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/service/DatasetServiceImpl.java

```diff
@@ -143,4 +150,29 @@ private Dataset updateDatasetValues(final Dataset datasetDb, final Dataset datas
 		return Utils.toList(repository.findByDatasetAcquisitionExaminationStudyId(studyId));
 	}
 
+	@Override
+	public List<Object[]> queryStatistics(String studyNameInRegExp, String studyNameOutRegExp, String subjectNameInRegExp, String subjectNameOutRegExp) throws Exception {
```

Comment:
Seems weird to me for this method to be in a Service and not a Repository class.
And little indentation / "login" comment pb :) (maybe extract a constant / Class for procedure name)

Reply:
as the web GUI is used, and normally the controller goes to the service, I think this is fine for me